packages/opentelemetry-cloud-monitoring-exporter/README.md

```diff
@@ -20,7 +20,15 @@ const { MeterProvider, PeriodicExportingMetricReader } = require("@opentelemetry
 const { MetricExporter } = require("@google-cloud/opentelemetry-cloud-monitoring-exporter");
 
 // Create MeterProvider
-const meterProvider = new MeterProvider();
+const meterProvider = new MeterProvider({
+  // Create a resource. These resources attributes will be translated to generic_task
+  // monitoring resource labels in Cloud Monitoring
+  resource: new Resource({
```

Comment:
should the example use resource detectors?

Reply:
Probably. Let me update

Reply:
Will add it once https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/563 is merged